clientGuide.md

```diff
@@ -0,0 +1,100 @@
+HTTP Wrapper
+============
+
+The [Bloomberg Open API] provides access to market data. The
+[Bloomberg Open API HTTP Wrapper] makes the API available via HTTP: clients
+can POST commands to an HTTP server running the wrapper, which will respond 
+with the relevant data.
+
+The HTTP wrapper is currently in development, and does not yet provide access
+to all features of the API. Notably, it currently supports the
+Request/Response paradigm of the API, but not Subscription.
```

Comment:
"supports the reference data service (i.e., /blp/refdata)"

I wouldn't mention what it doesn't support unless we are going to enumerate all the services note supported.